corehq/apps/analytics/static/analytix/js/hubspot.get_demo.js

```diff
@@ -0,0 +1,41 @@
+/* global window, hbspt */
+$(function () {
+    'use strict';
+
+    hqImport("analytix/js/hubspot").then(function () {
+        var hubspotGet = hqImport('analytix/js/initial').getFn('hubspot'),
+            hubspotApiId = hubspotGet('apiId');
+
+        // Old (control) hubspot form:
+        hbspt.forms.create({
+            portalId: hubspotApiId,
+            formId: "0f5de42e-b562-4ece-85e5-cfd2db97eba8",
+            target: "#get-demo-cta-form-body",
+            css: "",
+        });
+
+        // New (variant A) hubspot form:
+        hbspt.forms.create({
+            portalId: hubspotApiId,
+            formId: "38980202-f1bd-412e-b490-f390f40e9ee1",
+            target: "#get-demo-cta-form-content",
+            css: "",
+            onFormSubmit: function ($form) {
+                $('#get-demo-cta-calendar-content').fadeIn();
+                $('#get-demo-cta-form-content').addClass('hide');
+                var email = $form.find('[name="email"]').val(),
+                    firstname = $form.find('[name="firstname"]').val(),
+                    lastname = $form.find('[name="lastname"]').val(),
+                    newUrl = document.location.href + '?email=' + email + '&name=' + firstname + '%20' + lastname;
+
+                window.history.pushState(
+                    "dimagi-contact-url " + document.title, document.title, newUrl
```

Comment:
What's the first param here meant to be? [pushState docs](https://developer.mozilla.org/en-US/docs/Web/API/History_API#The_pushState()_method) say it should be an object, and our other `pushState` calls all just use null or an empty object.

Reply:
Also, I'm not quite following the overall flow and why this is added to history. So that once they're in scheduleonce/drift, they can click back and get to the filled-out form? Do they have to leave HQ to schedule a demo, or are scheduleonce/drift just widgets that pop up on the page?

Reply:
it's needed so that schedule once can populate the `email` and `name` fields. The first param is supposed to be a unique ID.